package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@apostrophecms/blog",
+  "version": "1.0.0",
+  "description": "Blog module for Apostrophe 3 websites",
+  "main": "index.js",
+  "scripts": {
+    "lint": "npm run eslint",
```

Comment:
I don't see an eslintrc, so I'm not sure if our rules are actually being applied here.

Reply:
Added file .eslintrc